project-minimization-plugin/src/main/kotlin/org/plan/research/minimization/plugin/services/MinimizationService.kt

```diff
@@ -20,22 +22,23 @@ class MinimizationService(project: Project, private val coroutineScope: Coroutin
 
     fun minimizeProject(project: Project) =
         coroutineScope.async {
```

Comment:
You can use this refactoring:
```suggestion
    fun minimizeProject(project: Project, parentContext: CoroutineContext = EmptyCoroutineContext) =
        coroutineScope.async(parentContext) {
```

And in place of call this:
```kotlin
minimizationService.minimizeProject(openedProject, coroutineContext).await()
```